source/Ship.h

```diff
@@ -260,13 +264,21 @@ class Ship : public Body, public std::enable_shared_from_this<Ship> {
 	// Get this ship's custom swizzle.
 	int CustomSwizzle() const;
 
+	void SetActivity(const std::string& name);
+	const std::string& GetActivity() const;
+
 	// Check if the ship is thrusting. If so, the engine sound should be played.
 	bool IsThrusting() const;
 	bool IsReversing() const;
 	bool IsSteering() const;
 	// The direction that the ship is steering. If positive, the ship is steering right.
 	// If negative, the ship is steering left.
 	double SteeringDirection() const;
+	// Set and count incoming projectiles.
+	int CountIncoming() const;
+	void SetIncoming(int count = 0);
+	int CountTargeting() const;
+	void SetTargeting(int count = 0);
```

Comment:
```suggestion

	// Set and count incoming projectiles.
	int CountIncoming() const;
	void SetIncoming(int count = 0);
	int CountTargeting() const;
	void SetTargeting(int count = 0);

```
just for readabillity